favorit/v1_views.py

```diff
@@ -0,0 +1,89 @@
+from django.shortcuts import get_object_or_404
+from favorit.forms import FavoritForm
+from restoran.models import Restoran
+from favorit.models import Favorit
+from makanan.models import Makanan
+from minuman.models import Minuman
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+import json
+
+def api_show_favorit(request):
+    """
+    API endpoint to show the list of favorites for the logged-in user.
+    """
+    favorit = Favorit.objects.filter(user=request.user)
+    favorit_list = [
+        {
+            "id": fav.id,
+            "makanan": fav.makanan.nama if fav.makanan else None,
+            "minuman": fav.minuman.nama if fav.minuman else None,
+            "restoran": fav.restoran.nama if fav.restoran else None,
+            "catatan": fav.catatan,
+        }
+        for fav in favorit
+    ]
+    return JsonResponse({"favorit": favorit_list}, status=200)
```

Comment:
```json
{
    "pk": "e3d95ba8-2b55-4214-a1c6-41b6527a31da",
    "fields": {
      "catatan": "",
      "user": 13,
      "makanan": null,
      "minuman": null,
      "restoran": "e9e54d55-1361-4e83-b880-aa7852ef16f8"
    }
  }
```

contoh response application/json untuk favorit, berseuaian dengan hasil dari django.core.serializers

Reply:
OK

Reply:
```json
[
    {
        "pk": "59db5b3c-bebe-43d7-9692-f78c3020256d",
        "fields": {
            "catatan": "",
            "user": 13,
            "makanan": null,
            "minuman": "052e46d2-d529-4f15-a912-6e57dfdf758c",
            "restoran": null
        }
    },
    {
        "pk": "07aac16c-9600-436d-a723-563e05e879fc",
        "fields": {
            "catatan": "",
            "user": 13,
            "makanan": "d758afc4-0162-4e1b-9490-cad603b5cf2a",
            "minuman": null,
            "restoran": null
        }
    }
]
```

Contoh response List of application/json, cukup application/json di dalam List.

Kalau yang dibuat itu adalah application/json bukan List of application/json.

```json
{
    "favorit": [
        {
            "pk": "59db5b3c-bebe-43d7-9692-f78c3020256d",
            "fields": {
                "catatan": "",
                "user": 13,
                "makanan": null,
                "minuman": "052e46d2-d529-4f15-a912-6e57dfdf758c",
                "restoran": null
            }
        },
        {
            "pk": "07aac16c-9600-436d-a723-563e05e879fc",
            "fields": {
                "catatan": "",
                "user": 13,
                "makanan": "d758afc4-0162-4e1b-9490-cad603b5cf2a",
                "minuman": null,
                "restoran": null
            }
        }
    ]
}
```

kenapa pakai yang atas? karena ngikutin hasil dari django.core.serializers